src/components/Reports/GoalCalculator/CalculatorSettings/Steps/SettingsStep/SettingsStep.tsx

```diff
@@ -0,0 +1,243 @@
+import React from 'react';
+import { Box, Container, Typography, styled } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+
+const StyledMainContainer = styled(Box)(({ theme }) => ({
+  padding: theme.spacing(1),
+  [theme.breakpoints.up('sm')]: {
+    padding: theme.spacing(1.5),
+  },
+  [theme.breakpoints.up('md')]: {
+    padding: theme.spacing(2),
+  },
+}));
+
+const StyledSectionContainer = styled(Box)(({ theme }) => ({
+  marginBottom: theme.spacing(1),
+  [theme.breakpoints.up('sm')]: {
+    marginBottom: theme.spacing(1.5),
+  },
+  [theme.breakpoints.up('md')]: {
+    marginBottom: theme.spacing(2),
+  },
+}));
+
+const StyledSectionTitle = styled(Typography)(({ theme }) => ({
+  flex: '1 1 100%',
+  fontSize: '0.875rem',
+  marginBottom: theme.spacing(1),
+  [theme.breakpoints.up('sm')]: {
+    fontSize: '1rem',
+  },
+  [theme.breakpoints.up('md')]: {
+    fontSize: '1rem',
+    marginBottom: 0,
+  },
+}));
+
+const StyledBoxContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  justifyContent: 'center',
+  marginBottom: theme.spacing(2),
+  flexWrap: 'wrap',
+  flexDirection: 'column',
+  alignItems: 'center',
+  gap: theme.spacing(1),
+  [theme.breakpoints.up('sm')]: {
+    justifyContent: 'flex-start',
+    flexDirection: 'row',
+    alignItems: 'flex-start',
+    gap: 0,
+    maxWidth: 'fit-content',
+  },
+}));
+
+const StyledSettingBox = styled(Box)(({ theme }) => ({
+  width: '140px',
+  height: '140px',
+  border: '1px solid',
+  borderRadius: theme.spacing(1),
+  borderColor: theme.palette.mpdxGray.main,
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  margin: theme.spacing(1),
+  padding: theme.spacing(1),
+  cursor: 'pointer',
+  '&:hover': {
+    borderColor: theme.palette.mpdxBlue.main,
+    boxShadow: '0 2px 8px rgba(0,0,0,0.1)',
+  },
+  [theme.breakpoints.up('sm')]: {
+    width: '160px',
+    height: '160px',
+    margin: theme.spacing(1.5),
+    padding: theme.spacing(1.5),
+  },
+  [theme.breakpoints.up('md')]: {
+    width: '200px',
+    height: '200px',
+    margin: theme.spacing(2),
+    padding: theme.spacing(2),
+  },
+}));
+
+const StyledImageBox = styled(Box)<React.ComponentProps<'img'>>(
```

Comment:
I believe this does what you want. Then when you use this component, remove the `component="img"` prop.

```suggestion
const StyledImage = styled('img')(
```